ui/src/album/AlbumDetails.js

```diff
@@ -49,6 +49,7 @@ const AlbumDetails = ({ classes, record }) => {
           {record.songCount}{' '}
           {translate('resources.song.name', { smart_count: record.songCount })}{' '}
           · <DurationField record={record} source={'duration'} />
+          · <SizeField record={record} source="size" />
```

Comment:
You should add a `{' '}` between the two fields, as you can see from your screenshot that the `·` is too close to the Duration time